app/views/records/_filters.html.haml

```diff
@@ -1,14 +1,16 @@
 %body
-
-  .filter-container.menu.large-3.columns
-    %header
+  %div.clearfix
+    %button.filter-btn.button.menu{value: 0} Filters
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace